testsuite/gnat2goto/tests/constant_declarations/test.adb

```diff
@@ -0,0 +1,9 @@
+with Const_Decs; use Const_Decs;
+
+procedure Test is
+   My_P : Integer := 4;
+begin
+   Inc (My_P);
+   --  Will fail: symex does not see the value
+   pragma Assert (My_P = 50);
```

Comment:
With this one I'm confused. Is this supposed to happen or not? Because I'd think after `Inc (My_P)` `My_P` would be 5, not 50?

Reply:
Perhaps the name for Inc is inappropriate. The subprogram actually adds a number based on the variable Var in the constant_decs package.  Using the value that Var is initialised with, Inc adds 46 to its parameter.  I think a more usual scenario that I think we should test is when a private type and deferred constant is used (as in the test below.

Reply:
Renamed to `Add_46`.